addons/mail/static/src/model/model_field_command.js

```diff
@@ -9,32 +9,44 @@ odoo.define('mail/static/src/model/model_field_command.js', function (require) {
 class FieldCommand {
     /**
      * @constructor
-     * @param {function} func function to call when executing this command. 
-     * The function should ALWAYS return a boolean value 
-     * to indicate whether the value changed.
+     * @param {string} name command name.
+     * @param {Object|Object[]} [data]
+     * data object or data objects array used for the command.
```

Comment:
This docstring is not correct, it can basically be any type `@param {*}`. For example `increment` has a number and not an object or a list.